contrib/postgres_fdw/deparse.c

```diff
@@ -3116,7 +3116,7 @@ deparsePartialAggFunctionParamFilter(Aggref *node, deparse_expr_cxt *context,
 			break;
 		case 2104:
 			/* float4 AVG function */
-			/* Fall through */
+			FALL_THROUGH
```

Comment:
Changes in contrib/postgres_fdw/deparse.c are redundant, because case follows case without any code between them